src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -66,6 +66,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <EmbeddedResource Include="..\..\..\LICENSE.txt" />
```

Comment:
I guess we may have to use a different LICENSE file for this nuget.exe as per https://github.com/NuGet/Client.Engineering/issues/2354#issuecomment-1610306380 comment.

Reply:
Ignore this comment, it looks like it is intentional to use repo root LICENSE file as per https://github.com/NuGet/Client.Engineering/issues/2354#issuecomment-2148069944